README.rst

```diff
@@ -27,10 +27,12 @@ Prerequisites
 
 * Install the ``bigframes`` package.
 * Create a Google Cloud project and billing account.
-* When running locally, authenticate with application default credentials. See
-  the `gcloud auth application-default login
-  <https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login>`_
-  reference.
+* In an interactive environment (like Notebook, Python REPL or command line),
+  ``bigframes`` will do the authentication on-the-fly if needed. Otherwise,
+  pre-authenticate by
+  `Installing and initializing the gcloud CLI <https://cloud.google.com/sdk/docs/install>`_,
```

Comment:
I would actually recommend linking to our Application Default Credentials docs for instructions. https://cloud.google.com/docs/authentication/provide-credentials-adc

Steps will vary depending on their environment. For example, if running on a GCE VM they won't have to do anything, either.

Reply:
Thanks for the suggestion, done, PTAL.